packages/stores/src/ui-config/manage-liquidity/add-liquidity.ts

```diff
@@ -576,4 +608,73 @@ export class ObservableAddLiquidityConfig extends ManageLiquidityConfigBase {
       );
     }
   }
+
+  @action
+  setConliqModalView = (
+    viewType: "overview" | "add_manual" | "add_managed"
+  ) => {
+    this._conliqModalView = viewType;
+  };
+
+  get conliqModalView(): "overview" | "add_manual" | "add_managed" {
+    return this._conliqModalView;
+  }
+
+  @action
+  setConliqHistoricalRange = (range: PriceRange) => {
+    this._conliqHistoricalRange = range;
+  };
+
+  get conliqHistoricalRange(): PriceRange {
+    return this._conliqHistoricalRange;
+  }
+
+  @action
+  setConliqMinRange = (min: Dec | number) => {
+    this._conliqRange = [
+      typeof min === "number" ? new Dec(min) : min,
+      this._conliqRange[1],
+    ];
+  };
+
+  @action
+  setConliqMaxRange = (max: Dec | number) => {
+    this._conliqRange = [
+      this._conliqRange[0],
+      typeof max === "number" ? new Dec(max) : max,
+    ];
+  };
+
+  get conliqRange(): [Dec, Dec] {
+    return this._conliqRange;
+  }
+
+  @action
+  setConliqFullRange = (isFullRange: boolean) => {
+    this._conliqFullRange = isFullRange;
+  };
+
+  get conliqFullRange(): boolean {
+    return this._conliqFullRange;
+  }
+
+  @action
+  setConliqBaseDepositAmountIn = (amount: Dec | number) => {
+    this._conliqBaseDepositAmountIn =
+      typeof amount === "number" ? new Dec(amount) : amount;
+  };
+
+  @action
+  setConliqQuoteDepositAmountIn = (amount: Dec | number) => {
+    this._conliqQuoteDepositAmountIn =
+      typeof amount === "number" ? new Dec(amount) : amount;
+  };
+
+  get conliqBaseDepositAmountIn(): Dec {
+    return this._conliqBaseDepositAmountIn;
+  }
+
+  get conliqQuoteDepositAmountIn(): Dec {
+    return this._conliqQuoteDepositAmountIn;
+  }
```

Comment:
It seems this could belong in it's own add liquiity config; something like `ObservableAddConcLiquidityConfig` like you had with the `AddConcLiquidityModal`, perhaps it can also extend `ManageLiquidityBase`???

Would help separat the logic a big, as it seems to not be dependent on this `ObservableAddLiquidityConfig` class in this file at all